packages/schematics/angular/workspace/files/tsconfig.json.template

```diff
@@ -3,11 +3,8 @@
   "compilerOptions": {
     "baseUrl": "./",
     "outDir": "./dist/out-tsc",<% if (strict) { %>
-    "noImplicitAny": true,
-    "noImplicitReturns": true,
-    "noImplicitThis": true,
-    "noFallthroughCasesInSwitch": true,
```

Comment:
`noFallthroughCasesInSwitch` should be kept.  It is not part of the `strict` option.

Reply:
Right!, fixed.